lib/common.py

```diff
@@ -329,9 +351,10 @@ def yum_remove_added_packages(self, base_packages):
             print("No added packages to remove.")
 
     def yum_restore_saved_state(self):
-        """ Currently only able to remove added packages """
-        assert self.saved_packages_list is not None, "Can't restore previous state without a package list"
-        self.yum_remove_added_packages(self.saved_packages_list)
+        """ Restore yum state to saved state """
+        assert self.saved_packages_list is not None and self.saved_rollback_id, "Can't restore previous state without a package list"
```

Comment:
I'd rather have two different asserts, one for package list and one for rollback id, each with its own message.